plugins/tiledb/io/TileDBReader.cpp

```diff
@@ -145,30 +112,31 @@ void TileDBReader::prepared(PointTableRef table)
 
 void TileDBReader::initialize()
 {
-    if (!m_cfgFileName.empty())
+    if (!m_args->m_cfgFileName.empty())
     {
-        tiledb::Config cfg(m_cfgFileName);
+        tiledb::Config cfg(m_args->m_cfgFileName);
         m_ctx.reset(new tiledb::Context(cfg));
     }
     else
         m_ctx.reset(new tiledb::Context());
 
     try
     {
-        if (m_stats)
+        if (m_args->m_stats)
             tiledb::Stats::enable();
 
 #if TILEDB_VERSION_MINOR < 15
```

Comment:
As this is a refactor, can we tighten this up, we will go to a version 3 of TileDB at some point.